ansible/testbed-cli.sh

```diff
@@ -743,9 +743,9 @@ function deploy_topo_with_cache
   echo "Done!"
 }
 
-vmfile=veos
+vmfile=ceos
```

Comment:
The vmfile must still be `veos`. This is the inventory file containing neighbor definitions, not neighbor type.